.vscode/tasks.json

```diff
@@ -37,11 +37,12 @@
             }
         },
         {
-            "label": "npm: buildBrowser",
+            "label": "buildBrowserDebug",
             "type": "npm",
-            "script": "buildBrowser",
-            "dependsOn": ["watch"],
-            "detail": "Builds the webpacked browser bundle"
+            "script": "buildBrowserDebug",
+            "detail": "Builds the webpacked browser bundle for debuggin",
+            // Not sure why if we really need this, but it was in the guide.
+            "problemMatcher": ["$ts-webpack-watch", "$tslint-webpack-watch"]
```

Comment:
I had this earlier but removed it because it adds a step to developer setup (which would need to be documented). See https://github.com/aws/aws-toolkit-vscode/commit/20f1ab62d302e9f3783f2e362be9919d361e7d8f